dotnet/src/SemanticKernel.Core/Functions/DelegatingAIServiceSelector.cs

```diff
@@ -18,8 +18,8 @@ internal class DelegatingAIServiceSelector : IAIServiceSelector
     internal AIRequestSettings? RequestSettings { get; set; }
 
     /// <inheritdoc/>
-    public (T?, AIRequestSettings?) SelectAIService<T>(string renderedPrompt, IAIServiceProvider serviceProvider, IReadOnlyList<AIRequestSettings>? modelSettings) where T : IAIService
+    public (T?, AIRequestSettings?) SelectAIService<T>(SKContext context, ISKFunction skfunction) where T : IAIService
```

Comment:
nit: sk**F**unction or just function

Reply:
`function` is a reserved keyword so I get this warning
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1716